soc/arm/st_stm32/stm32f7/dtcm.ld

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
So far STM32F7 is indeed the only STM32 series, though for coherency (and "futureproofness"), please move this file under `st_stm32/common`.

Reply:
Done.